SerializationTest is failing on windows

Part of #1338 

SerializationTest is failing after the latest commits.

[ERROR] Failures:
[ERROR]   SerializationTest$MultiLineStringsSerializedToScalarYamlBlocks.whenStringEndingWithNewline_thenAddBlockIndicatorInSerializedObject:240

I would like to be assigned for the issue since I have already done some research.

--- a/jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/SerializationTest.java
+++ b/jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/SerializationTest.java
@@ -224,12 +224,12 @@ class SerializationTest {
   class MultiLineStringsSerializedToScalarYamlBlocks {
     @Test
     @DisplayName("string ends with newline, then add | to use block style in serialized object")
-    void whenStringEndingWithNewline_thenAddBlockIndicatorInSerializedObject(@TempDir Path targetDir) throws IOException {
+    void unmarshal_whenStringEndingWithNewline_thenAddBlockIndicatorInSerializedObject(@TempDir Path targetDir) throws IOException {
       // Given
       final File targetFile = targetDir.resolve("cm.yaml").toFile();
       final ConfigMap source = new ConfigMapBuilder()
               .withNewMetadata()
-              .addToAnnotations("proxy.istio.io/config", String.format("proxyMetadata:%n    ISTIO_META_DNS_CAPTURE: \"false\"%nholdApplicationUntilProxyStarts: true\n"))
+              .addToAnnotations("proxy.istio.io/config", "proxyMetadata:\n    ISTIO_META_DNS_CAPTURE: \"false\"\nholdApplicationUntilProxyStarts: true\n")
               .endMetadata()
               .build();
       // When

